ws-nextjs-app/middleware.page.ts

```diff
@@ -6,13 +6,13 @@ import cspHeaderResponse from './utilities/cspHeaderResponse';
 
 const LOCALHOST_DOMAINS = ['localhost', '127.0.0.1'];
 
-export function middleware(request: NextRequest) {
+export async function middleware(request: NextRequest) {
```

Comment:
This change was made to allow `cspHeaderResponse()` to make a call to the the toggles API.